pkg/cmd/admin/policy/cani.go

```diff
@@ -79,6 +83,7 @@ func NewCmdCanI(name, fullName string, f *clientcmd.Factory, out io.Writer) *cob
 	cmd.Flags().StringVar(&o.User, "user", o.User, "Check the specified action using this user instead of your user.")
 	cmd.Flags().StringSliceVar(&o.Groups, "groups", o.Groups, "Check the specified action using these groups instead of your groups.")
 
+	kcmdutil.AddPrinterFlags(cmd)
```

Comment:
This adds a number of additional flags (`--sort-by`, `--no-headers`, etc). Do they all behave correctly?